binding.gyp

```diff
@@ -85,13 +85,19 @@
                         'WARNING_CFLAGS':[
                             '-Wno-c++11-narrowing',
                             '-Wno-constant-conversion'
+                        ],
+                        'LD_RUNPATH_SEARCH_PATHS': [
+                          '/opt/local/lib',
+                          '/usr/local/opt/curl/lib',
+                          '/usr/local/lib',
+                          '/usr/lib'
                         ]
                     },
                     'include_dirs' : [
                         '<!@(node "<(module_root_dir)/tools/curl-config.js" --cflags | sed s/-I//g)'
                     ],
                     'libraries': [
-                        '<!@(node "<(module_root_dir)/tools/curl-config.js" --libs)'
+                        '-L <!@(node "<(module_root_dir)/tools/curl-config.js" --prefix)/lib -lcurl'
```

Comment:
@koskokos2 hi! I'm looking at some changes again, can you explain why that was changed? Was it causing issues with the linking on macOS?

Reply:
As far as I remember previous way caused prebuilt libs on Travis to have irrelevant dependencies when only -lcurl is needed.

Reply:
makes sense, I'm adding support for static linking on linux and will need to put that back in, but will do so only on linux, will keep like the above for mac.

Thanks!

Reply:
Oh, that's amazing feature I was planning to start working on it myself in close future. Thanks for doing it!